### DISCUSSION TITLE
Reducing verbosity of generated CSS Modules class names

### DISCUSSION THREAD

**Discussion by wongmjane** [vercel/next.js#11768] [Help]
Hi! I noticed the generated of CSS module class names include the filename of the CSS file, even in production, for example:

<img width="540" alt="Screen Shot 2020-04-09 at 12 13 11 PM" src="https://user-images.githubusercontent.com/1332975/78857603-6c40fe00-7a5c-11ea-9b2c-b30147e1f69c.png">

I wonder if there’s anything I can do/config to prevent "Refractor" nor "comment/keyword/etc" from showing up in the generated class names. The filename nor the original class name are not something I hoped to be exposed in production. Thanks!

**Comment by Timer** [comment]
Happy to accept a PR minifying these class names!
